### PR TITLE
MET-2012: Credentials feedback

### PR DESCRIPTION
**Pull request by adamof**
✂️ Trims whitespace
✅ Validates credentials
🌀 Adds development setup (instructions TBD)

===

Jira story [#MET-2012](https://metrilojira.atlassian.net/browse/MET-2012) in project *Features team*:

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -14,6 +14,9 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
 
     const MODULE_NAME = 'Metrilo_Analytics';
 
+    public $js_domain = 't.metrilo.com';
```
Comment (evgeni-spasov-metrilo) [inline Helper/Data.php]: Is this used anywhere?

File: development/env [hunk 2]
```diff
@@ -0,0 +1,20 @@
+MYSQL_HOST=db
+MYSQL_ROOT_PASSWORD=myrootpassword
+MYSQL_USER=magento
+MYSQL_PASSWORD=magento
```
Comment (evgeni-spasov-metrilo) [inline development/env]: Let's use the username and password we agreed for development docker services:
```
user: metrilo
password: metrilo1
```
I'm changing the wordpress ones currently.

Comment (adamof) [reply development/env]: 👍

File: Helper/Data.php (hunk 1 above)
Comment (adamof) [reply Helper/Data.php]: ah, forgot to use it, yes 👍

**Comment by adamof** [comment]
Addressed the requested changes. Sorry for not doing them in separate commit.
The backslashes are something from php (PSR)

File: development/env (hunk 2 above)
Comment (evgeni-spasov-metrilo) [reply development/env]: Remind, remind.

**Comment by nadamova** [comment]
tested 👍

**Pull request merged**